### PR TITLE
Update 2.1 Alpine 3.6 images to 3.7

### PR DESCRIPTION
**Pull request by MichaelSimons**
As part of this I am proposing the introduction of fixed `alpine3.7` suffixed tags along with the stable `alpine` tags.  This aligns with what is done in the official Docker repos.  This will afford us the opportunity to upgrade Alpine versions within any given .NET release.

Related to #488

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,8 +88,8 @@ After the application starts, navigate to `http://localhost:8000` in your web br
 
 # Linux arm32 tags
 
-- [`2.0.6-runtime-stretch-arm32v7`, `2.0-runtime-stretch-arm32v7`, `2.0.6-runtime`, `2.0-runtime`, `2-runtime`, `runtime` (*2.0/runtime/stretch/arm32v7/Dockerfile*)](https://github.com/dotnet/dotnet-docker/blob/nightly/2.0/runtime/stretch/arm32v7/Dockerfile)
```
Comment (MichaelSimons) [inline README.md]: I can't explain why this is showing up as a diff.  I didn't change anything here - see latest nightly https://github.com/dotnet/dotnet-docker/blob/nightly/README.md

**Comment by richlander** [comment]
LGTM

Once we ship RTM, I assume we'll support a given alpine tag until it goes EOL. Yes?

**Comment by MichaelSimons** [comment]
@richlander - yes.

**Comment by MichaelSimons** [comment]
@dotnet-bot test NanoServer-sac2016 - 2.1 Dockerfiles please

**Pull request merged**